code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -460,12 +460,12 @@ <h2>Table of Symbols</h2>
                   <td><em>\(\frac{\text{m}}{\text{s}}\)</em></td>
                 </tr>
                 <tr>
-                  <td><em><b>v</b>(t)<sub>f</sub><sup>AB</sup></em></td>
+                  <td><em><b>v</b>(t)<sup>AB</sup><sub>f</sub></em></td>
```

Comment:
one of the reasons for the huge change is this: sub and sup have changed order. That seems weird - why would they? I would track that down to stabilize this.

Reply:
Oh, I did that intentionally. The order doesn't necessarily matter to LaTeX, so I just made a decision based on my usual way.